Error in msapproval task: failed to find transaction: pending transaction 1 not found

### Describe the bug:
msapproval task error: failed to find transaction: pending transaction 1 not found:
https://github.com/filecoin-project/lily/blob/00875522c545dc6e609bc2bece1f92b030ca05f4/tasks/msapprovals/msapprovals.go#L240-L257

### Steps to Reproduce:

### Lily Version: v0.8.5

sentinel archiver hit the same error 2022-01-27T12:07:55.060Z        INFO    archiver        sentinel-archiver/verify.go:146 error found for height 1493195: [{"Addr":"f2unf6ajcfrxis6wcehnwws2aocb46k5qn3u7ta3q","Error":"failed to find transaction: pending transaction 1 not found"}]    {"task": "msapprovals", "export_file": "/mnt/disk1/archiver/rawcsv/mainnet/arch0127-2022-01-25-visor_processing_reports.csv"}

--- a/tasks/msapprovals/msapprovals.go
+++ b/tasks/msapprovals/msapprovals.go
@@ -71,7 +71,7 @@ func (p *Task) ProcessMessages(ctx context.Context, ts *types.TipSet, pts *types
 			continue
 		}
 
-		applied, tx, err := p.getTransactionIfApplied(ctx, m.Message, m.Receipt, pts)
+		applied, tx, err := p.getTransactionIfApplied(ctx, m.Message, m.Receipt, ts)
 		if err != nil {
 			errorsDetected = append(errorsDetected, &MultisigError{
 				Addr:  m.Message.To.String(),

